doc/Command-Reference.md

```diff
@@ -7961,8 +7962,305 @@ Go Back To [Beginning of the document](#) or [Beginning of this section](#waterm
+
+- Usage:
+  ```
+  Usage: sonic-package-manager repository remove [OPTIONS] NAME
+
+    Remove package from database.
+
+  Options:
+    --help  Show this message and exit.
+  ```
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-package-manager repository remove cpu-report
+  ```
+
+**sonic-package-manager install**
+
+This command pulls and installs package on SONiC host. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager install [OPTIONS] [PACKAGE_EXPR]
+
+    Install package using [PACKAGE_EXPR] in format "<name>==<version>" or "<name>@<tag|digest>"
+
+  Options:
+    --enable                      Set the default state of the feature to
+                                  enabled and enable feature right after
+                                  installation. NOTE: user needs to execute
+                                  "config save -y" to make this setting
+                                  persistent
+    --default-owner [local|kube]  Default owner configuration setting for a
+                                  feature
+    --from-repository TEXT        Fetch package directly from image registry
+                                  repository NOTE: This argument is mutually
+                                  exclusive with arguments: [package_expr,
+                                  from_tarball].
+    --from-tarball FILE           Fetch package from saved image tarball NOTE:
+                                  This argument is mutually exclusive with
+                                  arguments: [package_expr, from_repository].
+    -f, --force                   Force operation by ignoring failures
+    -y, --yes                     Automatically answer yes on prompts
+    -v, --verbosity LVL           Either CRITICAL, ERROR, WARNING, INFO or DEBUG
+    --skip-host-plugins           Do not install host OS plugins provided by the
+                                  package (CLI, etc). NOTE: In case when package
+                                  host OS plugins are set as mandatory in
+                                  package manifest this option will fail the
+                                  installation.
+    --help                        Show this message and exit.
+  ```
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-package-manager install dhcp-relay==1.0.2
+  ```
+  ```
+  admin@sonic:~$ sudo sonic-package-manager install --from-repository azure/sonic-cpu-report:latest
+  ```
+  ```
+  admin@sonic:~$ sudo sonic-package-manager install --from-tarball sonic-docker-image.gz
+  ```
+
+**sonic-package-manager uninstall**
+
+This command uninstalls package from SONiC host. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager uninstall [OPTIONS] NAME
+
+    Uninstall package
```

Comment:
```suggestion
    Uninstall package
```

Reply:
Should we also allow uninstalling multiple packages at once?

Reply:
It can be added